scrapper/nomad_list_scrapper.py

```diff
@@ -85,19 +108,22 @@ def get_cities(self):
         Takes the cities from the home page, builds a dictionary for each one with the available information.
         Then, returns a list of dicts with all the cities.
         """
-        cities = []
+        self._get_html()
+        self._get_all_the_cities()
+        self._make_request_to_city_details()
 
-        for res in grequests.map(self._make_request_to_city_details(), size=CFG.NOMAD_LIST_REQUESTS_BATCH_SIZE,
-                                 exception_handler=self._exception_handler):
+        for res in self._cities_details:
```

Comment:
I'm not a big fan of having effect in our class. It could bring us some problems with concurrency for example.
In this case, I think, that it's the same, and we won't have big problems.